### PR TITLE
Careplan auth

### PR DESCRIPTION
**Pull request by daliboz**
Apply templates and authorization to careplan

Minor cleanup of a few other links/text.

### CODE REVIEW

File: content/millennium/dstu2/care-provision/care-plan.md [hunk 1]
```diff
@@ -7,10 +7,51 @@ title: CarePlan | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The DSTU 2 CarePlan resource can be used to represent both care plans and care teams.
```
Comment (MaxPhilips) [inline content/millennium/dstu2/care-provision/care-plan.md]: Page title has DSTU 2 with a space but other places in the text have it without

File: content/millennium/dstu2/care-provision/care-plan.md [hunk 2]
```diff
@@ -7,10 +7,51 @@ title: CarePlan | DSTU 2 API
...
+
+### Care Plans
+
+The CarePlan resource describes the intentions of how one or more practitioners plan to deliver care to a specific patient. Care plans are used in a variety of areas and scopes ranging from encounter or visit specific to cross-visits.  Care plans are used to group activities, goals and/or practitioners to create context. Care plans are intended to be specific to the patient. It is possible for a planned activity to stand alone without a care plan such as a scheduled appointment.  
+
+The following fields are returned if valued:
+
+* [Id](http://hl7.org/fhir/dstu2/resource-definitions.html#Resource.id){:target="_blank"}
```
Comment (MaxPhilips) [inline content/millennium/dstu2/care-provision/care-plan.md]: CarePlan id / CareTeam id below

File: content/millennium/dstu2/care-provision/care-plan.md [hunk 3]
```diff
@@ -7,10 +7,51 @@ title: CarePlan | DSTU 2 API
...
+* [Context (patient encounter)](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.context){:target="_blank"}
+* [Date/Time created](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.period){:target="_blank"}
+* [Author](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.author){:target="_blank"}
+* [Date/Time last modified](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.modified){:target="_blank"}
+* [Category (assess-plan)](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.category){:target="_blank"}
+* [Care plan](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.relatedPlan.plan){:target="_blank"}
+
+### Care Teams
+
+The CareTeam resource will be available in FHIR STU3. Meanwhile, the CarePlan resource will be used to identify the care team members in DSTU2. Care team members or participants include practitioner (physicians, nurses, technicians, etc.), family members, friends, guardians, and the patient. The care team can be specific to an encounter or the patient across all encounters (longitudinal).
```
Comment (MaxPhilips) [inline content/millennium/dstu2/care-provision/care-plan.md]: Add trademark for FHIR

File: content/millennium/dstu2/care-provision/care-plan.md (hunk 2 above)
Comment (MaxPhilips) [reply content/millennium/dstu2/care-provision/care-plan.md]: Never mind I see these are all switched to just Id :)

File: content/millennium/dstu2/care-provision/care-plan.md [hunk 4]
```diff
@@ -40,7 +87,7 @@ _Implementation Notes_
```
Comment (MaxPhilips) [inline content/millennium/dstu2/care-provision/care-plan.md]: Remove default/max _count values

**Comment by MaxPhilips** [comment]
+1

**Comment by daliboz** [comment]
Addressed comments in one way or another :)

File: content/millennium/dstu2/care-provision/care-plan.md [hunk 5]
```diff
@@ -7,10 +7,51 @@ title: CarePlan | DSTU 2 API
...
+
+The following fields are returned if valued:
+
+* [Id](http://hl7.org/fhir/dstu2/resource-definitions.html#Resource.id){:target="_blank"}
+* [Subject (patient only)](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.subject){:target="_blank"}
+* [Status](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.status){:target="_blank"}
+* [Context (patient encounter)](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.context){:target="_blank"}
+* [Time period](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.period){:target="_blank"}
+* [Last modified](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.modified){:target="_blank"}
+* [Category (careteam, Longitudinal, or encounter)](http://hl7.org/fhir/dstu2/careplan-definitions.html#CarePlan.category){:target="_blank"}
```
Comment (dennispatterson) [inline content/millennium/dstu2/care-provision/care-plan.md]: make 'Longitudinal' lower-case

**Comment by dennispatterson** [comment]
+1

File: lib/resources/example_json/dstu2_examples_care_plan.rb [hunk 6]
```diff
@@ -43,114 +43,89 @@ module Resources
...
-                "member": {
-                  "reference": "Practitioner/1912007",
-                  "display": "Smart, Aliza"
+        "resourceType": "Bundle",
+        "id": "4b27e0a6-6e5a-4785-9cc7-d4749388a518",
+        "type": "searchset",
+        "total": 1,
+        "link": [{
+            "relation": "self",
+            "url": "https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/CarePlan?patient=4734007\u0026category=careteam"
```
Comment (JFisk42) [inline lib/resources/example_json/dstu2_examples_care_plan.rb]: What's going on with this patient here? "patient=4734007\u0026"

Comment (JFisk42) [reply lib/resources/example_json/dstu2_examples_care_plan.rb]: Oh I see, it's an ampersand :)

**Pull request merged**